drag-sort-listview-gradle/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+
+// This file is here (rather than in the drag-sort-listview project)
+// so we can control how it builds without having to submit pull requests.
+
+dependencies {
+    compile 'com.android.support:support-v4:19.1.0'
+}
+
+android {
+    compileSdkVersion 7
```

Comment:
That's what the library was [using before](https://github.com/bauerca/drag-sort-listview/blob/master/library/project.properties#L15) and I didn't want to introduce any functional changes in this diff, so I was trying to keep everything the same.